pkgs/stdenv/cross/default.nix

```diff
@@ -1,10 +1,10 @@
 { lib
-, system, platform, crossSystem, config
+, localSystem, crossSystem, config
```

Comment:
Ah, OK, this makes sense! Do you think there's somewhere you can document this nomenclature?